src/main/java/com/automation/school/ad_main/Collection/UserOrder.java

```diff
@@ -0,0 +1,80 @@
+package com.automation.school.ad_main.Collection;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserOrder  implements Order{
+
+    private long id; //(generate automatically in constructor )
+    public LocalDateTime date; //(set as current day directly in constructor)
+    public User user;
+    public List<Item> items;
+    private static int nextUniqueId = 0;
+
+
+    // public  User newUser;
+    //public  Item newItem;
+    //get the value of id
+    public UserOrder() {
+        this.id = getId();
+        this.date = LocalDateTime.now();
+        this.items = new ArrayList<>();
+        //this.user = user;
+    }
+
+    public int getId() {
+        nextUniqueId++;
+        id = nextUniqueId;
+        return (int) id;
+    }
+
+    @Override
```

Comment:
why do all 3 methods return zeroes and nulls both it this class and in CompanyOrder?
if you don't need them - remove from interface, and you can get rid of overriding those here

Reply:
methods count and LocalDateTime   was deleted.
sum is modified to return  Price sum. but usage is not implemented. will be done soon.